custom_components/bosch_alarm/alarm_control_panel.py

```diff
@@ -20,14 +21,21 @@
 READY_STATE_HOME = 'home'
 READY_STATE_AWAY = 'away'
 FAULTED_POINTS_ATTR = 'faulted_points'
```

Comment:
Unrelated to this PR, but I noticed that the faulted_points count doesn't actually update... Haven't had a chance to look into this yet.

Reply:
oh wait i was thinking of something else, i haven't actually looked to see if faulted points works for me

Reply:
however as youve said, its unrelated to this PR, so maybe we raise a seperate issue about that to keep track